site/hugo.yaml

```diff
@@ -66,7 +66,7 @@ params:
   active_releases:
     # Mention all active releases here, in semver descending order
     - "1.0.0"
```

Comment:
we don't have `1.0.0` release yet, right? Should we delete that?

Reply:
We can keep it. It only renders if the release is present on the versioned-docs branch.